optimum/intel/openvino/modeling_base_seq2seq.py

```diff
@@ -66,7 +66,7 @@ def __init__(
         self.model_save_dir = kwargs.get("model_save_dir")
         self._device = kwargs.get("device", "CPU")
         self.is_dynamic = kwargs.get("dynamic_shapes", True)
-        self.ov_config = {"PERFORMANCE_HINT": "LATENCY"}
```

Comment:
Why is it more efficient to remove the `"PERFORMANCE_HINT"` key ?

Reply:
> Why is it more efficient to remove the `"PERFORMANCE_HINT"` key ?

Jan is on holiday so I'll answer for now: great question! It is not expected, but this showed improved performance. We suspect that the latency performance hint was optimized for single models, not encoder/decoder models. We will look into this more.

Reply:
@echarlaix exactly as @helena-intel stated. It will be consulted with OV teams, as this case is very interesting one.

PS While I am on holidays, I am still receiving notifications from open source -- I will answer occasionally:)

Reply:
Thank you for the explanation !